catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -115,7 +123,8 @@ public String getWkt() throws CswException {
         // started
         reader.moveUp();
 
-        LOGGER.debug("Returning WKT in LON/LAT coord order: {}.", wkt);
+        wkt = convertToEPSG4326(wkt, crs);
```

Comment:
I think this should check the `cswAxisOrder` and only call that conversion method if it's in lat/lon.